src/models/client-session-actions-request.ts

```diff
@@ -7,7 +7,12 @@ interface IClientSessionActionsRequest {
 type ClientSessionAction = IClientSessionAction;
 
 interface IClientSessionAction {
-  type: 'SET_PAYMENT_METHOD' | 'UNSET_PAYMENT_METHOD';
+  type: 'SET_PAYMENT_METHOD' | 'UNSET_PAYMENT_METHOD' | 'SET_BILLING_ADDRESS';
```

Comment:
Shouldn't this be `SELECT_PAYMENT_METHOD` and `UNSELECT_PAYMENT_METHOD` for alignment?